dev-libs/libqtxdg/libqtxdg-3.0.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils virtualx versionator
+
+DESCRIPTION="A Qt implementation of XDG standards"
+HOMEPAGE="http://lxqt.org/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxde/${PN}.git"
+else
+	SRC_URI="https://github.com/lxde/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="0/$(get_version_component_range 1)"
+IUSE="test"
+
+CDEPEND="
+	dev-qt/qtcore:5=
+	dev-qt/qtdbus:5=
+	dev-qt/qtgui:5=
+	dev-qt/qtsvg:5=
+	dev-qt/qtwidgets:5=
+	dev-qt/qtxml:5=
```

Comment:
have you spoken to the Qt team (@kensington) whether you are allowed to put `:=` on these? `:=` on qt deps is only allowed in cases of packages consuming private APIs.

Reply:
No, I did not. Yeah, in ideal world it should work as you say, and public API should be stable, no bugs should appear. In the real world, however, it does not work that way, and sometimes some strange bugs appear while they should not. For example, see: https://github.com/gentoo/gentoo/pull/5920#issuecomment-336519793

I don't want to hit issues like that if it can be avoided with minor additional cost.

Reply:
@iDarkTemplar so they think commits like https://github.com/gentoo/gentoo/commit/3a97b1edecd630c0f48d3b0a28f053b0489a9597 are totally fine, but this isn't? Lol